numpy/random/_bounded_integers.pyx.in

```diff
@@ -51,16 +51,6 @@ cdef extern from "numpy/random/distributions.h":
                                   np.npy_bool *out) nogil
 
 
-
-_integers_types = {'bool': (0, 2),
```

Comment:
Did nothing use these values?

Reply:
I checked it, and I don't see any references. 
Please check: https://github.com/numpy/numpy/search?q=_integers_types&unscoped_q=_integers_types